cli/cmd/logs/logs.go

```diff
@@ -63,6 +64,9 @@ func (l *Logs) setupConfig(ctx *clicontext.CLIContext) (*stern.Config, error) {
 		if obj.Object == nil {
 			return nil, errors.New("No object found")
 		}
+		if obj.Object.(*riov1.Service).Status.DeploymentReady == true && obj.Object.(*riov1.Service).Status.ScaleStatus != nil && obj.Object.(*riov1.Service).Status.ScaleStatus.Available == 0 {
```

Comment:
Why do we need to throw an error? Is there a use case that we can wait for pod to be created? Feel like it is not really needed. If service has zero pod just log nothing cause there is no pod.

Reply:
708 was made to bring inline with `exec`, but I agree it could go either way. It's slightly confusing because there is no output, it looks like command is hanging. Perhaps we could log "no current pods to log" or something.

Reply:
I just think there is a use case to log service from the very beginning and wait for pods to be created. If this change is made then you can log service only if there are pods that already exist.

Reply:
Yes, lets print and wait then: `Waiting for pods....` and then continue logging.

Reply:
updated